helix-core/src/main/java/org/apache/helix/HelixManagerFactory.java

```diff
@@ -19,19 +19,18 @@
  * under the License.
  */
 
-/**
- * factory that creates cluster managers
- *
- * for zk-based cluster managers, the getZKXXX(..zkClient) that takes a zkClient parameter
- *   are intended for session expiry test purpose
- */
 import org.apache.helix.manager.zk.HelixManagerStateListener;
 import org.apache.helix.manager.zk.ZKHelixManager;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+
 /**
  * Obtain one of a set of Helix cluster managers, organized by the backing system.
+ * factory that creates cluster managers
+ *  *
+ *  * for zk-based cluster managers, the getZKXXX(..zkClient) that takes a zkClient parameter
+ *  *   are intended for session expiry test purpose
```

Comment:
This description is not quite easy to follow.

Reply:
@zhangmeng916 

Objectively, this comment is not very helpful to me - one tip when giving reviews & comments: either make your comment concrete by clarifying what's not easy to follow, or suggest an alternative?

With that said, I'm just going to remove the second block since it doesn't apply here.